progenitor-impl/tests/output/test_freeform_response.out

```diff
@@ -0,0 +1,59 @@
+pub use progenitor_client::{ByteStream, Error, ResponseValue};
```

Comment:
These are expected output for test comparison purposes?

Reply:
Exactly right; this one is used here:
https://github.com/oxidecomputer/progenitor/pull/35/files#diff-086b5af8b29e4521f8cdd3158911231ad150c262d4507f197397e7f6a0bb6a2fR102-R105